types/mongodb/index.d.ts

```diff
@@ -2566,7 +2567,7 @@ export class GridFSBucketReadStream extends Readable {
 
 /** https://mongodb.github.io/node-mongodb-native/3.1/api/GridFSBucketReadStream.html */
 export interface GridFSBucketReadStreamOptions {
-    sort?: number;
+    sort?: SortValues;
```

Comment:
This one is a bit problematic, please see https://github.com/mongodb/node-mongodb-native/pull/2519

I'd like to wait for some feedback at MongoDB driver before calling it, but I believe the correct type here should be `Array<[string, SortValues]> | SortOptionObject<T>`.

Reply:
Agreed. So this would be both a bug fix and a type narrowing at this point vs just a type narrowing right?

Reply:
Based on https://github.com/mongodb/node-mongodb-native/pull/2519#issuecomment-677714844 I suggest to include the above fix, then this can be merged.

Reply:
```suggestion
    sort?: Array<[string, SortValues]> | SortOptionObject<T>;
```

Reply:
@HitkoDev - let me know if that's what you intended?

If not then you can use the "Suggest Change" feature in the comment.